terraform/ec2/win/main.tf

```diff
@@ -112,10 +111,12 @@ resource "null_resource" "integration_test_reboot" {
   count = length(regexall("restart", var.test_dir)) > 0 ? 1 : 0
 
   connection {
-    type     = "winrm"
-    user     = "Administrator"
-    password = rsadecrypt(aws_instance.cwagent.password_data, local.private_key_content)
-    host     = aws_instance.cwagent.public_dns
+    type            = "ssh"
+    user            = "Administrator"
+    password        = rsadecrypt(aws_instance.cwagent.password_data, local.private_key_content)
+    host            = aws_instance.cwagent.public_ip
+    target_platform = "windows"
+    timeout         = "6m"
```

Comment:
is 6m arbitrary? I don't really care either way, was just curious.

Reply:
It's from this PR: https://github.com/aws/amazon-cloudwatch-agent/pull/579/files
(The default is 5m)[https://developer.hashicorp.com/terraform/language/resources/provisioners/connection#argument-reference], so I would assume that extra 1m was found to be useful with PR #579.